### ISSUE TITLE
Can we support adding custom headers for authentication?

### ISSUE THREAD

**Issue by kbirger** [kwoding/wiremock-rest-client#13]
WireMock.org advertises a cloud offering of wiremock via MockLab.io. This supports all of the same APIs as basic WireMock (as well as some additional ones), but requires an API key for auth.

```bash
curl --output my-stubs.json \
  -H 'Authorization:Token my-api-token' \
  https://my-api.mocklab.io/__admin/mappings
```

Can this package be extended to support passing this header? The most general way to do this would be to allow passing of a collection of headers in the constructor of the client in the form of an associative object:

```typescript
const headers = { 
  Authorization: 'Token my-api-token'
};

const wireMock = new WireMockRestClient('http://localhost:8080', headers);
```

This would be a great improvement. If there's support, I can put out a PR for this.

**Comment by kwoding** [comment]
Just opened a pull request that should add this support, could you please check/test?

**Comment by kbirger** [comment]
Hey, thanks for the speedy response. I'll take a look!

**Comment by kbirger** [comment]
I tried this out after changing the types to match your example. I'm not able to get it to work. I can dig in more later, but it doesn't look like the `headers` are getting sent across.

**Comment by kwoding** [comment]
Fixed on the branch, just released version 1.9.0 with this functionality
